Creating a basic landing page to which we can link from our outbound … (#30988)
gatsbyjs/gatsby: docs/docs/reference/release-notes/v2-maintenance-plan.md

---
title: Gatsby Framework Version Support
---

> ## Overview

Reviewer: nit: swapping this out to "Introduction" to match the template (https://www.gatsbyjs.com/contributing/docs-contributions/how-to-write-a-reference-guide/)

```suggestion
## Introduction
```

Author: Make sense! Consistency is key!

As we continue evolving the Gatsby framework, and as we move from one major version to the next, we will communicate the duration and types of support provided to the prior major version. This allows users to effectively plan their Gatsby framework upgrades.

## What is happening, when?

> Gatsby Version 2.32 will enter Maintenance Long Term Support on **June 8, 2021**. At that same time, Gatsby Version 3 will be considered the Active Long Term Support version of Gatsby. The Maintenance Long Term Support will continue until the release of Gatsby v4 (date TBD).

Reviewer: Reorganizing the paragraph to try and group things together. Feel free to make adjustments.

It might also be worth it to briefly explain the difference between "maintenance long-term support" and "active long-term support". What's the difference in terms of what users can expect from us for each case?

```suggestion
Starting on **June 8, 2021**:

* Gatsby Version 3 will be considered the Active Long-Term Support version of Gatsby.
    * This means that ???.
* Gatsby Version 2.32 will enter Maintenance Long-Term Support.
    * This means that ???.
    * The Maintenance Long-Term Support will continue until the release of Gatsby v4 (date TBD).
```

Author: Yeah I love that, much more readable, and better structure. Will fill in the blanks. Thanks @meganesu

## What does this mean to users?

> - As of June 8, 2021, Gatsby version 2.32 will only receive critical patches.

Reviewer: This feels like it needs another sentence to explain what the implications are for the user.

e.g. "Gatsby 2.32 will only receive critical patches, which means that we won't be accepting new issues for that version." (or whatever the boundaries are.) How will the user's normal behavior patterns be affected?

Reviewer: It might also be worth it to say something like "if you have a problem with v3, you can still open an issue." (Although that might fit better in the "critical patches" section below.)

Author: Good points. Never hurts to reassure and remind that / where issues can be reported.

Author: I added callouts here to issues / support area of .com as well.

Also, since I explain in the `What is happening, when` section that v3 is required for feature enhancements, I didn't see the need to reiterate that v2 wouldn't be receiving new features.

> - To take advantage of new capabilities within Gatsby, you must be on version 3.0 or later

Reviewer: This could be a good place to link to the v3 migration guide

Author: Good call - added.
